static/templates/settings/bot-settings.handlebars

```diff
@@ -7,6 +7,14 @@
             {{/tr}}
         </p>
 
+        <div class="input-group">
+            <label for="type_of_bot">{{t "Type of Bot" }}:</label>
+            <select name="type_of_bot" id="user_type_of_bot">
+                <option value="{{ custom_bot }}">Custom Bot</option>
+                <option value="{{ incoming_webhook_bot }}">Incoming Webhook Bot</option>
+            </select>
+        </div>
```

Comment:
Yes, it does! I am working on JS changes, just wanted to make sure if this is what is expected.